frontend/src/components/program/FormScheduleEntryItem.vue

```diff
@@ -8,7 +8,7 @@
           :name="$tc('components.program.formScheduleEntryItem.start')"
           vee-id="startDate"
           vee-rules="required"
-          :allowed-dates="allowedStartDates"
+          :max="localScheduleEntry.start"
```

Comment:
Not sure I understand this logic:
- I can pick a date that is maximum the start date of the scheduleEntry? 
- before with allowedDates, the user was able to select any valid date within a period. With min/max this restriction will not be possible


**Before**
![image](https://user-images.githubusercontent.com/449555/205656658-206a3830-d16a-482d-ad24-978db7f164aa.png)


**Now**
![image](https://user-images.githubusercontent.com/449555/205657162-bc2c2936-bb97-4202-9744-4e7e90d39f8e.png)

Reply:
Uhm, yes, that is a bug, thanks. Should day `:max="localScheduleEntry.end"`. As you said on the other comment, we can improve this if we implement the common behaviour of having the end date move when the start date is moved. But on the end date picker it should still say `:min="localScheduleEntry.start"`. I will implement that and write tests for this component.

And yes, I can additionally restrict the date picker to the periods in this form. I didn't know we support moving schedule entries across periods. As long as we support this, this is another blocker for #2454. I have now added it there.

Reply:
Fixed in [2ebe629](https://github.com/ecamp/ecamp3/pull/3172/commits/2ebe629ab3e8bac8756b26a7393185fb01f6f496)